evap/results/templates/result_widget.html

```diff
@@ -0,0 +1,37 @@
+{% load results_templatetags %}
+{% load evaluation_filters %}
+
+{% spaceless %}
+    {% if question_result.has_answers %}
+        <div class="d-flex{% if question_result.warning %} participants-warning{% endif %}"
+            data-toggle="tooltip" data-placement="left"
+            title="{% if question_result.warning %}{% trans 'Only a few participants answered this question.' %}</br></br>{% endif %}
+            {% for count, name in question_result.counts|zip:question_result.choices.names %}
+                {{ name }}: {{ count }}/{{ question_result.count_sum }} ({{ count|percentage_one_decimal:question_result.count_sum }})
+                {% if not forloop.last %}</br>{% endif %}
+            {% endfor %}"
+        >
+            {% if not single_result %}
+                <div class="badge-participants mr-3">
+                    <span class="fas fa-user small"></span> {{ question_result.count_sum }}
```

Comment:
`small` sets `font-weight: 400`, overwriting the rule from `fas`, resulting in another icon.